AFQ/data/fetch.py

```diff
@@ -20,15 +20,16 @@
 import time
 
 import bids.config as bids_config
+import warnings
 try:
+    warnings.simplefilter(action='ignore', category=FutureWarning)
     bids_config.set_option('extension_initial_dot', True)
```

Comment:
Remind me what `extension_initial_dot` does?

Do I understand correctly that this will stop working in pybids 0.16? Or is this config simply not needed anymore at that point?